Sources/Publish/API/Path.swift

```diff
@@ -11,8 +11,9 @@ import Foundation
 public struct Path: StringWrapper {
     public var string: String
 
-    public init(_ string: String) {
+    public init(_ string: String, absolute: Bool = false) {
```

Comment:
Your problem seems to be that there is no property declaration for "absolute" . Its only mentioned in the constructor

Reply:
@jlorfeo Right... I should have caught that. Downsides to coding when tired haha. Thanks so much!